src/Administration/Resources/app/administration/src/docs/README.md

```diff
@@ -0,0 +1,48 @@
+# Administration Internal Architecture Docs (Draft Skeleton)
+
+> Purpose: Internal-facing technical documentation for the Shopware 6 Administration (Vue application). Each linked document currently contains only a bullet list of intended content. Expand iteratively.
+
+## Sections
+
+- Overview (`01-overview/`)
+- Architecture (`02-architecture/`)
+- Extensibility (`03-extensibility/`)
+- Data Layer (`04-data-layer/`)
+- Runtime (`05-runtime/`)
```

Comment:
Just an idea about this "Runtime" category. Because I don't really know what this mean until I explicitly look into it. Maybe rename it to "Globals" or "Global objects"?.

And then putting `03-context-auth` under `09-security`?

```
05-globals-objects/
    01-global-shopware-object
    02-feature-flags

09-security/
    ...
    ...
    03-context-auth
```